src/terminal/writer.go

```diff
@@ -163,9 +163,11 @@ func Pwd(pwdType, userName, hostName, pwd string) string {
 	}
 
 	if strings.HasSuffix(pwd, ":") {
-		pwd += "\\"
+		pwd += `/`
 	}
 
+	pwd = strings.ReplaceAll(pwd, `\`, `/`)
```

Comment:
On Linux/macOS, the backslash (`\`) is a valid character in filenames. In that case it will produce a wrong path.